### ISSUE TITLE
Add alt-text to website/docs images

### ISSUE THREAD

**Issue by garrettjstevens** [GMOD/jbrowse-components#1451]
A lot of the images in our docs look like this, with no alt-text (the text that goes inside the square brackets):

```
![](./img/some_img.png)
```

The alt-text is important for accessibility, e.g. for people who use screen readers. The alt-text should describe what is in the image so that someone who cannot see the image could still benefit.

One note, there seems to be some parsing of alt-text in `tex_parser.js` that I don't totally understand.

**Comment by cmdcolin** [comment]
Every picture in our docs should have a figure caption below the figure.

The tex parser changes that line that is below a figure into the ![caption here](pic) form, and it becomes a nice caption. Could probably comment that a bit better, but really it would be more ideal if the markdown just had a better caption system

**Comment by garrettjstevens** [comment]
I think that makes more sense with what's happening in `tex_parser.js`. So pandoc renders the alt-text in the square brackets as a caption?

My concern, then, is that the text in the brackets is alt-text needed for accessibility, and with that system, none of the individual pages will have the alt-text, only the combined page (as I understand it).

**Comment by cmdcolin** [comment]
>I think that makes more sense with what's happening in tex_parser.js. So pandoc renders the alt-text in the square brackets as a caption?

Ya, so the tex_parser behavior and such should be pdf only

I guess my idea was to have captions always because this serves as readable instructions that transcends a screen reader, it is just instructive for all contexts. In that sense, I don't know if we need the alt-text?

**Comment by cmdcolin** [comment]
The addition of the mdx <Picture/> element may have helped with this
